sdk/python/arvados/arvfile.py

```diff
@@ -516,7 +517,7 @@ def _block_prefetch_worker(self):
                     return
                 self._keep.get(b)
             except Exception:
-                pass
+                _logger.error(traceback.format_exc())
```

Comment:
@colin-nolan "`_logger.exception()` does include a stack trace" is true when called from an exception handler (as in ["This method should only be called from an exception handler."](https://docs.python.org/2/library/logging.html#logging.Logger.exception) :)

```python
import logging
logging.basicConfig()
_logger = logging.getLogger()
try:
    nan = 1/0
except:
    _logger.exception("boom")
```

```
ERROR:root:boom
Traceback (most recent call last):
  File "boom.py", line 5, in <module>
    nan = 1/0
ZeroDivisionError: integer division or modulo by zero
```